lib/template/config/config.rb

```diff
@@ -1,5 +1,9 @@
 require "pliny/config_helpers"
 
+
+# Supress the "use RbConfig instead" warning.
+Object.send(:remove_const, :Config) if Object.const_defined?(:Config)
```

Comment:
Moved this from `lib/pliny/config_helpers.rb` because we need to be able to require that helper without declaring the `Config` module.